### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by FsDevNinja**
added twitter bootstrap, simple form, and devise

### CODE REVIEW

**Comment by kellylougheed** [comment]
It all looks good to me, although I'm not sure about the mailers -- it might be good to link to our actual heroku app in production.rb (up-all-knight.herokuapp.com). Also for environment.rb, I've linked to localhost:3030 previously -- not sure if that's terribly different from localhost, port: 3000.

File: config/environments/development.rb [hunk 1]
```diff
@@ -1,37 +1,37 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (lasley) [inline config/environments/development.rb]: Could always overload the controller to do this dynamically. Something like:

```
class ApplicationController < ActionController::Base
  def default_url_options
    { host: request.host_with_port }
  end
end
```

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '4c3651fc174bd5f93f1beb6c031cc1066cd019adb04ab30d6a4e957002a609fe6bf764f7cfa9d9518c22265a1f88d7bd4e8088c1f862f40deafbb4d21b8b285f'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (lasley) [inline config/initializers/devise.rb]: changeme?

File: spec/factories.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
 FactoryGirl.define do
+  factory :player do
+
```
Comment (lasley) [inline spec/factories.rb]: Might be good to fill in some default attrs here

File: config/environments/development.rb (hunk 1 above)
Comment (kellylougheed) [inline config/environments/development.rb]: This looks like a nice solution to be compatible with everyone's localhost.

**Comment by lasley** [comment]
@FsDevNinja - Merging your own code is bad practice. Additionally, there was only one explicit review acceptance here & a lot of code that wasn't looked at by anyone on the team.

**Comment by FsDevNinja** [comment]
noted.

**Comment by GonSanchezS** [comment]
@lasley is it possible to un-merge? Just curious for the future.

**Comment by lasley** [comment]
Anything is possible, but this would involve rewriting history. It's a major no-no to rewrite history on a production branch (master) though, because it will create a de-sync with anyone else that has the code checked out.

**Pull request merged**